scripts/register-completions.bash

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# Parameter completion for the dotnet CLI
+
+_dotnet_bash_complete()
+{
+  local word=${COMP_WORDS[COMP_CWORD]}
+  local dotnetPath=${COMP_WORDS[1]}
+
+  local completions=("$(./bin/Debug/netcoreapp1.0/osx.10.11-x64/publish/dotnet complete --position ${COMP_POINT} "${COMP_LINE}")")
```

Comment:
In the near term I'll replace these paths with an unqualified path which will allow this to work as long as dotnet is on your path. In the future, the script will provide the path to `dotnet` as a parameter so that multiple side-by-side installs will work correctly in the event that the available completions differ among them.